Description: Use platform library under libcec-platform name.
 The library is renamed for packaging it in Debian under a less generic name.
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,16 +6,16 @@
 
 find_package(kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
-find_package(platform REQUIRED)
+find_package(cec-platform REQUIRED)
 find_package(ZLIB REQUIRED)
 
 include_directories(${kodiplatform_INCLUDE_DIRS}
-                    ${platform_INCLUDE_DIRS}
+                    ${cec-platform_INCLUDE_DIRS}
                     ${KODI_INCLUDE_DIR}
                     ${ZLIB_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}/lib)
 
-set(DEPLIBS ${platform_LIBRARIES}
+set(DEPLIBS ${cec-platform_LIBRARIES}
             ${ZLIB_LIBRARIES})
 
 message(STATUS "ZLIB_LIBRARIES: ${ZLIB_LIBRARIES}")
